src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  align-self: center;
+}
+
+.logo img {
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/232076007-c2f6b205-ef29-4d6c-a216-0d3b75520245.png)

Reply:
not fixed